app/views/users/_user.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="user_card">
+  <div class="user_img my_border">
+    <img src="<%= user.photo %>" width="100" height="100" alt="Profile picture">
+  </div>
+  <div class="user_info my_border">
+    <h2 class="user_name">
+      <%= user.name %>
+    </h2>      
+    <p class="post_count">Number of posts: <%= user.posts_counter %></p>     
+  </div>
+</div>
```

Comment:
- [ ] Consider Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)